### ISSUE TITLE
ls -la crashes with thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)', src/libcore/result.rs:1084:5

### ISSUE THREAD

**Issue by mrusme** [ogham/exa#728]
```
▲  RUST_BACKTRACE=1 ls -la                                                                                                                                                                                                             thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)', src/libcore/result.rs:1084:5
stack backtrace:
   0: std::panicking::default_hook::{{closure}}
   1: std::panicking::rust_panic_with_hook
   2: std::panicking::continue_panic_fmt
   3: rust_begin_unwind
   4: core::panicking::panic_fmt
   5: core::result::unwrap_failed
   6: exa::output::table::Table::display
   7: <F as scoped_threadpool::FnBox>::call_box
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
[1]    52323 abort      RUST_BACKTRACE=1 exa --git -la
 △  RUST_BACKTRACE=full ls -la                                                                                                                                                                                                          thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)', src/libcore/result.rs:1084:5
stack backtrace:
   0:        0x105e2dced - std::panicking::default_hook::{{closure}}::hffc69ca72fa98b80
   1:        0x105e2d7bc - std::panicking::rust_panic_with_hook::h058ef423beec40ce
   2:        0x105e2df7d - std::panicking::continue_panic_fmt::h0bb2cdb45a553955
   3:        0x105e2e249 - rust_begin_unwind
   4:        0x105ebfdfa - core::panicking::panic_fmt::h2413d97d26203710
   5:        0x105ec0399 - core::result::unwrap_failed::hef33208e3f23e07a
   6:        0x105df36bd - exa::output::table::Table::display::h7d61868c2e1aa281
   7:        0x105e06955 - <F as scoped_threadpool::FnBox>::call_box::hfeff2cdcb76e7a57
   8:        0x105e1ed6e - std::sys_common::backtrace::__rust_begin_short_backtrace::hf388fc36537a8fe9
   9:        0x105e1f89e - core::ops::function::FnOnce::call_once{{vtable.shim}}::hd49318dcebc5ef76
  10:        0x105e3813d - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::hf778f932793a5a82
  11:        0x105e38767 - std::sys::unix::thread::Thread::new::thread_start::ha4ccb7099fa6037f
  12:     0x7fff6a29a109 - _pthread_start
[1]    52421 abort      RUST_BACKTRACE=full exa --git -la
```

```
ls -v
exa v0.9.0
```

**Comment by toxinu** [comment]
This has been fixed on `master` but the fix is not part of the `v0.9.0` packages.
Could you create a new release @ogham?

**Comment by avanier** [comment]
I had the same crash on the `v0.9.0` Fedora package. Building from master _(built at 78ba0b8973cbfbb48b85b8beae6f7de412064d11)_ fixed it for me.

Refer to #687 which seems to aggregate a lot of the duplicates of this issue.

**Comment by ogham** [comment]
Yeah, the fix for this got merged in #670. When there is a new release it will go out.

Soon, I promise!
